### PR TITLE
HMAN- 50

### PR DESCRIPTION
**Pull request by padmapriyanalam**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/HMAN-50

### Change description ###

Update Database with PUT request data

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/hmc/helper/CaseHearingRequestMapperTest.java [hunk 1]
```diff
@@ -44,7 +45,6 @@ void modelToEntity() {
         assertEquals("CMLC123", entity.getOwningLocationId());
         assertEquals(Boolean.FALSE, entity.getCaseRestrictedFlag());
         assertEquals(Boolean.FALSE, entity.getCaseRestrictedFlag());
-        assertEquals(1, entity.getVersionNumber());
```
Comment (markdathornehmcts) [inline src/test/java/uk/gov/hmcts/reform/hmc/helper/CaseHearingRequestMapperTest.java:47]: Are we ok losing this assert?

File: src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java [hunk 2]
```diff
@@ -352,16 +396,15 @@ private HearingEntity updateHearingStatusAndVersionNumber(Long hearingId, String
             if (null != hearingEntity.getCaseHearingRequest()) {
                 log.info("CHANGING: version number : {}", hearingEntity.getCaseHearingRequest().getVersionNumber());
                 hearingEntity.getCaseHearingRequest().setVersionNumber(
-                    hearingEntity.getCaseHearingRequest().getVersionNumber() + 1);
+                    hearingEntity.getCaseHearingRequest().getVersionNumber() + VERSION_NUMBER_TO_UPDATE);
```
Comment (markdathornehmcts) [inline src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java:399]: Interested in this new variable VERSION_NUMBER_TO_UPDATE

File: src/test/java/uk/gov/hmcts/reform/hmc/helper/CaseHearingRequestMapperTest.java (hunk 1 above)
Comment (padmapriyanalam) [reply src/test/java/uk/gov/hmcts/reform/hmc/helper/CaseHearingRequestMapperTest.java:47]: Code has been refactored to use same methods for PUT and POST methods. Version number is validated in IT tests.

File: src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java (hunk 2 above)
Comment (padmapriyanalam) [reply src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java:399]: just to avoid hard code variable  , nothing much :-)

**Review by markdathornehmcts — APPROVED** [review]

**Pull request closed without merging**